draft-ietf-quic-recovery.md

```diff
@@ -300,30 +300,24 @@ receiver.
 The alarm duration, or Probe Timeout (PTO), is set based on the following
 conditions:
 
-* If there is exactly one unacknowledged packet, PTO SHOULD be scheduled for
-  max(2*SRTT, 1.5*SRTT+kDelayedAckTimeout)
-
-* If there are more than one unacknowledged packets, PTO SHOULD be scheduled for
-  max(2*SRTT, 10ms).
+* PTO SHOULD be scheduled for max(1.5*SRTT+MaxAckDelay, 10ms)
 
 * If RTO ({{rto}}) is earlier, schedule a TLP alarm in its place. That is,
   PTO SHOULD be scheduled for min(RTO, PTO).
 
-kDelayedAckTimeout is the expected delayed ACK timer.  When there is exactly one
-unacknowledged packet, the alarm duration includes time for a delayed
-acknowledgment to be received by including kDelayedAckTimeout.
-
-The RECOMMENDED value for kDelayedAckTimeout is 25ms.
-
-(TODO: Add negotiability of delayed ack timeout.)
+MaxAckDelay is the maximum ack delay supplied in an incoming ack frame.
+MaxAckDelay excludes ack delays that aren't included in an RTT sample because
+they're too large and excludes those which reference an ack-only packet.
 
-A PTO value of at least 2*SRTT ensures that the ACK is overdue. Using a PTO of
-exactly 1*SRTT may generate spurious probes, and 2*SRTT is simply the next
-integral value of RTT.
+QUIC diverges from TCP by calculating MaxAckDelay dynamically, instead of
+assuming a constant value for all connections.  It includes this in all probe
```

Comment:
TCP doesn't assume a constant value for all connections. I would say something like "instead of implicitly including it in the SRTT and RTTVAR estimates".

Reply:
I state that below, but I updated this text to be a bit more specific.